apps/test/unit/lib/levelbuilder/script-editor/LessonTokenTest.jsx

```diff
@@ -34,4 +37,21 @@ describe('LessonToken', () => {
     expect(wrapper.contains('Lesson 1')).to.be.true;
     expect(wrapper.find('i').length).to.equal(3);
   });
+
+  it('renders newly added lesson without edit button', () => {
+    let wrapper = mount(
+      <LessonToken
+        {...defaultProps}
+        lesson={{
+          key: 'new-lesson',
+          name: 'New Lesson',
+          levels: [],
+          position: 1
+        }}
+      />
+    );
+    expect(wrapper.find('Motion').length).to.equal(1);
+    expect(wrapper.contains('New Lesson')).to.be.true;
+    expect(wrapper.find('i').length).to.equal(2);
```

Comment:
If it would be straight forward, I would suggest checking that `.fa-pencil` is not present here and is present in the default case.

Reply:
Done!